src/apps/dashboard/modal/still-in-queue-modal/queued-reservation-item.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { FC } from "react";
+import CheckBox from "../../../../components/checkbox/Checkbox";
+import { useText } from "../../../../core/utils/text";
+import { FaustId } from "../../../../core/utils/types/ids";
+import fetchDigitalMaterial from "../../../loan-list/materials/utils/digital-material-fetch-hoc";
+import fetchMaterial, {
+  MaterialProps
+} from "../../../loan-list/materials/utils/material-fetch-hoc";
+
+export interface QueuedReservationItemProps {
+  numberInQueue: number;
+  faust?: FaustId;
+  identifier?: string;
+  selectedReservations: {
+    [key: string]: string;
+  }[];
+  setCustomSelection: (elementId: string, reservationId: string) => void;
+  reservationId: string;
+}
+
+const QueuedReservationItem: FC<QueuedReservationItemProps & MaterialProps> = ({
+  material,
+  numberInQueue,
+  faust = "",
+  identifier = "",
+  selectedReservations,
+  setCustomSelection,
+  reservationId
+}) => {
+  const t = useText();
+  const { title, authors, year, materialType } = material || {};
+  return (
+    <li>
+      <div className="list-materials">
+        <div className="list-materials__checkbox mr-32">
+          <CheckBox
+            id={faust || identifier}
+            label=""
+            selected={Object.keys(selectedReservations).includes(
+              faust || identifier
+            )}
+            onChecked={() => {
+              setCustomSelection(faust || identifier, reservationId);
+            }}
+          />
+        </div>
+        <div className="list-materials__content">
+          <div className="list-materials__content-status">
+            <div className="status-label status-label--outline ">
+              {materialType}
+            </div>
+          </div>
+          <p className="text-header-h5 mt-8">{title}</p>
+          <p className="text-small-caption">
+            {authors && authors} {year && year}
```

Comment:
We have about a gazillion places now where we have this constructtion with an author an year. We should really make a component out of it. I respect if you dont feel you have time for that. But please write a todo about that we should move this to a common/global component